src/main/java/com/palehorsestudios/ard/Game.java

```diff
@@ -154,8 +157,9 @@ String Look(Player player, String option) {
     /**
      * Method to invoke unlock chest method
      */
-    void unlockChest(Player player) {
+    String unlockChest(Player player) {
         player.getCurrentRoom().unlockChest();
```

Comment:
Need to return the result of player.getCurrentRoom().unlockChest() in the returned String. You will have to modify unlockChest().